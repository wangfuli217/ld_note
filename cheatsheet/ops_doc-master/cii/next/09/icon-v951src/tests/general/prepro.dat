#  prepro.dat -- code fragment $included by prepro.icn

write("prepro.dat here")

$undef abc
$define abc 321
$define xyzzy 47
