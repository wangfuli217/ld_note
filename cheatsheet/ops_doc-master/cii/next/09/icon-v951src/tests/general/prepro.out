abc,def,ghi,xx,yy: 123 456 987 1 2
prepro.dat here
xyzzy: 47
abc,def,ghi,xx,yy: 321 456 987 1 2
found feature: ASCII
no feature:    EBCDIC
done
